specification/alertsmanagement/resource-manager/Microsoft.AlertsManagement/preview/2022-09-01-preview/SmartDetectorAlertRulesApi.json

```diff
@@ -0,0 +1,773 @@
+    },
+    "AlertRulesList": {
+      "type": "object",
+      "description": "List of Smart Detector alert rules.",
+      "properties": {
+        "value": {
+          "description": "List of Smart Detector alert rules.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AlertRule"
+          }
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to get the next set of results."
+        }
+      }
+    },
+    "AlertRulePatchObject": {
+      "properties": {
+        "id": {
+          "description": "The resource ID.",
+          "readOnly": true,
+          "type": "string"
+        },
+        "type": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The resource type."
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The resource name."
+        },
+        "tags": {
+          "type": "object",
+          "description": "The resource tags.",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/AlertRulePatchProperties",
+          "description": "The properties of the alert rule.",
+          "minProperties": 1
+        }
+      },
+      "type": "object",
+      "description": "The alert rule patch information"
+    },
+    "Detector": {
+      "properties": {
+        "id": {
+          "type": "string",
+          "description": "The detector id."
+        },
+        "parameters": {
+          "description": "The detector's parameters.'",
+          "type": "object",
+          "additionalProperties": {
+            "type": "object"
+          }
+        },
+        "name": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The Smart Detector name."
+        },
+        "description": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The Smart Detector description."
+        },
+        "supportedResourceTypes": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "type": "string"
+          },
+          "description": "The Smart Detector supported resource types."
+        },
+        "imagePaths": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "type": "string"
+          },
+          "description": "The Smart Detector image path. By default this is not populated, unless it's specified in expandDetector"
+        },
+        "parameterDefinitions": {
+          "type": "array",
+          "readOnly": true,
+          "description": "The Smart Detector parameters definitions.'",
+          "items": {
+            "$ref": "#/definitions/DetectorParameterDefinition"
+          },
+		  "x-ms-identifiers": []
```

Comment:
>		 [](http://example.com/codeflow?start=0&length=2)

nit : use spaces not tabs

Reply:
also, I am not sure what is causing the remaining errors. the errors and their explanations are not clear enough